data/samil-web.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Shamil Aliyev",
+  "type": "personal",
+  "bio": "Engineer turned into Programmer. In a journey to make remarkable things"
```

Also, add a missing comma `,` here. 

```suggestion
  "bio": "Engineer turned into Programmer. In a journey to make remarkable things",
```